ui/src/lib/components/object-browser/object-browser.css

```
@@ -0,0 +1,576 @@
+.ob-root {
+  position: fixed;
+  inset: 0;
+  z-index: 50;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Syne', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)